snf-cyclades-app/synnefo/userdata/migrations/0002_auto_20161026_0853.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+import datetime
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('userdata', '0001_initial'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='publickeypair',
+            name='created_at',
+            field=models.DateTimeField(default=datetime.datetime(2016, 10, 26, 8, 53, 18, 407755), auto_now_add=True),
+            preserve_default=False,
+        ),
+        migrations.AddField(
+            model_name='publickeypair',
+            name='deleted_at',
+            field=models.DateTimeField(default=None, null=True),
+            preserve_default=True,
+        ),
+        migrations.AddField(
+            model_name='publickeypair',
+            name='updated_at',
+            field=models.DateTimeField(default=None, null=True),
```

Comment:
I agree that there is an inconsistency in terms of model declaration/generated migration result. According to [OpenStack API](http://developer.openstack.org/api-ref/compute/?expanded=update-a-cell-detail,show-keypair-details-detail) this field can be nullable.